src/components/override-sdk/field/RichText/RichText.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import ParsedHTML from '../../../helpers/formatters/ParsedHtml';
+
+export default function RichText({ value }) {
```

Comment:
I think RichText is used in many places so it should not break anything. Can you please ask @IanOvenden approval here.

Reply:
This override component will only replace the OOTB component within the claimant portal, and is not something used extensively.  Within Constellation context (AUI) rich text components will remain unchanged.  Discuss with Nick and happy to proceed.